CHANGELOG.md

```diff
@@ -6,11 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## \[Unreleased\]
+
+## 17/03/2023 - Release 1.1.0rc1
 ### Added
```

Comment:
Can you use 
Added ->New features
Changed -> Enhancements
Fixed -> Bug fixes

Reply:
Updated it